build/tsc.gni

```diff
@@ -0,0 +1,47 @@
+import("npm.gni")
+
+template("typescript_build") {
+  assert(defined(invoker.tsconfig),
+         "Need tsconfig name to run (must exist in electron root)")
```

Comment:
what does `must exist in electron root` mean here? it looks like the path gets rebased so it'll be the absolute path, why does it matter where the tsconfig is?